CONTRIBUTING.md

```diff
@@ -1,3 +1,24 @@
+## Setting up your working environment
+
+If you want to contribute to the project you'll first need to install the dependencies.
+You can do it via opam:
+```
+$ git clone git@github.com:realworldocaml/mdx.git
```

Comment:
I think this is not necessary

Reply:
I think it is, for convenience. Say I just opened the contributing and want to follow the instructions but haven't cloned it yet, I just have to copy/paste that line without going back to the homepage, click 'Clone or download', clip the copy to clipboard, type `git clone` etc...